web_development_101/git_basics/project_practicing_git_basics.md

```diff
@@ -81,3 +81,5 @@ For now, concentrate on using the basics of Git that you've learned here with al
 This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something.
 
 * [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) by [Chris Beams](https://github.com/cbeams) provides guidelines for writing neat and concise commit messages.
+
+* [Learn Github in 20 Minutes](https://www.youtube.com/watch?v=nhNq2kIvi9s&t=952s) by [Colt Steele](https://www.youtube.com/channel/UCrqAGUPPMOdo0jfQ6grikZw) provides more knowledge on learning to use GIT and GitHub.
```

Comment:
First link has a timestamp, is this intentional?

I also think that linking the channel is redundant.